src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -0,0 +1,490 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Pane.h"
+
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Microsoft::Terminal::Settings;
+using namespace winrt::Microsoft::Terminal::TerminalControl;
+
+static const int SEPARATOR_SIZE = 4;

```

Comment:
Nit: We should avoid using uppercase names.  Pascal case is recommended.  Or is the plan to fix these when we do  #921?

Reply:
Yea I think I wrote this code originally before that was apparent. I'll fix this, since it's in new code